mlflow/protos/service.proto

```diff
@@ -591,7 +591,7 @@ service MlflowService {
   rpc getTrace (GetTrace) returns (GetTrace.Response) {
     option (rpc) = {
       endpoints: [{
-        method: "GET",
+        method: "POST",
```

Comment:
why do we need this change?

Reply:
If I was a user, I'd be surprised if `getTrace` uses `POST`. `_get_request_message` can't be fixed?